### DISCUSSION TITLE
Changing DialogTrigger tray placement to the right in RSP

### DISCUSSION THREAD

**Discussion by cheema-corellian** [adobe/react-spectrum#6393] [Q&A]
Hi there,

Our designer asked me to look into the possibility of having the dialog tray component slide from the right side, instead of the bottom.  I promptly tried the following:

<DialogTrigger type="tray" placement="right">

However, placement appears to be ignored for tray type DialogTrigger.  It always slides up from the bottom.  Is it correct to assume that it is not possible to have it slide in from the right?  The target device for our application is desktop and not mobile.

[Link to DialogTrigger tray storybook component](https://reactspectrum.blob.core.windows.net/reactspectrum/108bcecd84918c9f1f2d3d9032593237142cedde/storybook/index.html?path=/story/dialogtrigger--type-tray&args=placement:top&providerSwitcher-express=false&providerSwitcher-scale=large)

**Comment by snowystinger** [comment]
Correct, as per Spectrum, it only slides in from the bottom https://spectrum.adobe.com/page/tray/#Animation
